### PR TITLE
jeos tests

### PR DESCRIPTION
**Pull request by kravciak**

### CODE REVIEW

**Comment by aaannz** [comment]
you don't have to tidy manually, do as travis do:
`ln -s ../os-autoinst/tools; ./tools/tidy`

File: tests/jeos/firewall_enabled.pm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    my $check_ssh_port="grep 'FW_SERVICES_EXT_TCP=' /etc/sysconfig/SuSEfirewall2 | cut -d'\"' -f2";
+
+    validate_script_output "$check_state", sub { /^active$/ };
+    validate_script_output "$check_substate", sub { /^exited$/ };
+    validate_script_output "$check_ssh_port", sub { /^22$/ };
+}
+
+sub test_flags() {
+    return { 'important' => 1 };
+}
```
Comment (coolo) [inline tests/jeos/firewall_enabled.pm]: leave out those unneeded quotes

**Comment by coolo** [comment]
you misunderstood me. I was talking about
```
- return {'important' => 1};
+ return {important => 1};
```

**Comment by coolo** [comment]
and once you're done, please git rebase to squash these cleanup commits

File: tests/jeos/firewall_enabled.pm [hunk 2]
```diff
@@ -0,0 +1,20 @@
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub run() {
+    my $self           = shift;
```
Comment (coolo) [inline tests/jeos/firewall_enabled.pm]: and yet another nitpick: you don't need to set $self if you don't need it in the run function

**Comment by coolo** [comment]
and looking at your tests, it looks like we should make important the default - it would cut your code in half ;)

**Comment by coolo** [comment]
LGTM

**Pull request merged**